src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/viewmodel/RecordScriptureViewModel.kt

```diff
@@ -0,0 +1,103 @@
+package org.wycliffeassociates.otter.jvm.app.ui.takemanagement.viewmodel
+
+import com.github.thomasnield.rxkotlinfx.observeOnFx
+import io.reactivex.Observable
+import javafx.beans.property.SimpleBooleanProperty
+import javafx.beans.property.SimpleObjectProperty
+import javafx.beans.property.SimpleStringProperty
+import javafx.collections.ObservableList
+import org.wycliffeassociates.otter.common.data.model.ContentLabel
+import org.wycliffeassociates.otter.common.data.workbook.Chunk
+import org.wycliffeassociates.otter.jvm.app.ui.workbook.viewmodel.WorkbookViewModel
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
+import tornadofx.*
+
+class RecordScriptureViewModel : ViewModel() {
+    private val workbookViewModel: WorkbookViewModel by inject()
+    private val audioPluginViewModel: AudioPluginViewModel by inject()
+
+    val recordableViewModel = RecordableViewModel(audioPluginViewModel)
+
+    // This will be bidirectionally bound to workbookViewModel's activeChunkProperty
+    private val activeChunkProperty = SimpleObjectProperty<Chunk?>()
```

Comment:
does this need to be nullable?
properties can hold nulls, so this has like, two potential levels of nulls?

Reply:
I thought I commented on this, but maybe not. You're right, we didn't need this to be nullable, so I changed it to non-nullable.